src/main/java/org/fungover/mmotodo/user/User.java

```diff
@@ -40,17 +41,19 @@ public class User {
 
     @Column(name = "created")
     @CreationTimestamp
-    private LocalDateTime created;
+    private Instant created;
 
     @Column(name = "updated")
     @UpdateTimestamp
-    private LocalDateTime updated;
+    private Instant updated;
 
     @NotNull
     @ManyToOne(fetch = FetchType.LAZY, optional = false)
     @JoinColumn(name = "tasks", nullable = false)
     private Task tasks;
 
+
```

Comment:
Remove empty lines. Maybe is enough to revert this file to its latest commited state?